MimeKit/Utils/MimeUtils.cs

```diff
@@ -508,6 +508,8 @@ public static string Quote (string text)
 			return Quote (text.AsSpan ());
 		}
 
+		private static readonly char[] unquoteChars = new[] { '\r', '\n', '\t', '\\', '"' };
+
 		/// <summary>
```

Comment:
I prefer to define my "constants" at the top and using PascalCase.

Reply:
Updated.